### PR TITLE
Pop up

### PR DESCRIPTION
**Pull request by clin2on3mun**
pop up branch: Add pop-up modal when button see project clicked

- it uses HTML and CSS and javascript to achieve this event
- Uses CSS to style pages
- Javascript to give the button events

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,6 +1,158 @@
...
+}];
+
+const cardSectionContent = [{
+  cardTitles: 'Profesional Art Printing Data',
+  cardTitles1: ['Data Dashboard Healthcare', 'Website Portfolio', 'Proffesional Art Printing Data More'],
+  cardDescription: `A daily selection of privately personalized reads;
+  no accounts or sign-ups required. Has been the industry's standard.`,
+  cardButton: ['HTML', 'Bootstrap', 'Ruby', 'See project'],
+}];
+
```
Comment (bobb-Rob) [inline script.js]: - [ ] Based on the project requirement, you are expected to have an array of project details like so 👇🏾 

```javascript
const projects = [{
    id: 1,
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
},
{
    id: 2,
    name: "Ghost game",
    description: "A Ghost FPS game built with javascript.",
    image: "link_to_image.jpg",
    technologies: ["HTML", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
},
{
    id: 3,
    name: "CV Builder",
    description: "A CV builder which helps users .",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
},
{
    id: 4,
    name: "Sun city",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

This means that each project is displayed in the work's sections and when the see project button is clicked, the modal popup appears using that specific project data populated with the data from the projects array. Currently, when I click the see project button for any project card, I see the same information and this is not supposed to be. Check an example of students' projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95 and https://github.com/nicupop729/Portfolio/pull/11/commits/aabed37cc2208f52d0bd081a4cae8cdc83c49636.

Thanks to the above points, adding projects in the future will be easy. Students will only add it in one place - as another object in the projects array.

File: styles.css [hunk 2]
```diff
@@ -154,6 +154,78 @@ ul {
...
+    width: 95%;
+    top: 2%;
+    left: 2%;
+    bottom: 3%;
+    display: none;
+    overflow-y: scroll;
+    margin: 0 auto;
+
+    /* position:fixed; */
+    height: auto;
```
Comment (bobb-Rob) [inline styles.css]: - [ ] You did very well with styling the popup modal. Well done 👏🏾

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @clin2on3mun , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 3]
```diff
@@ -1,6 +1,234 @@
...
+  no accounts or sign-ups required. has been the industry's standard dummy
+  text ever since the 1500s, when an unknown printer took a standard dummy text.`,
+  recentWorkButton: ['CSS', 'HTML', 'Bootstrap', 'Ruby', 'See project'],
+  SeeLive: '',
+  seeSource: '',
+
+}];
+
+const cardSectionContent = [{
+  id: '1',
```
Comment (OyePriscilla) [inline script.js]: - [ ] Please as raised earlier and according to the project requirement, kindly use an array to store all the details of your project. 👍 

Untitled

File: script.js [hunk 4]
```diff
@@ -1,6 +1,234 @@
...
+  cardDescription: `A daily selection of privately personalized reads;
+  no accounts or sign-ups required. Has been the industry's standard.`,
+  modalDescription: `Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+  Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown okay sir
+  `,
+  cardButton: ['HTML', 'Bootstrap', 'Ruby', 'See project'],
+  SeeLive: '',
+  seeSource: '',
+  imageModal: 'images/feature_image.png',
+},
```
Comment (OyePriscilla) [inline script.js]: - [ ] Please kindly ensure that the corresponding details are displayed whenever the `see project button` is clicked. The previous reviewer pointed this out as well. 👍 **Note: I'm getting the same images for all the project cards.**

**Review by OyePriscilla — APPROVED** [review]
Hi @clin2on3mun and @AliShahSafdari 👏 👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Use JavaScript to manipulate DOM elements ✔️ 
Use JavaScript events ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [Optional] Please kindly format your README appropriately, kindly add these missing headers (Tech stack and acknowledgement); don't forget to remove he FAQ section if there is not FAQ at the moment ℹ️ Implementing all these will make your repo looks professional and more engaging. 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @clin2on3mun, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewers. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




### To Highlight :1st_place_medal: 

-  Descriptive Pull Request title and summary. ✔️
- User clicks the button, the popup with details about the project appears ✔️
- User clicks the close (X) button, the popup disappears ✔️
- Modal popup data is populated with the data from the projects array ✔️
- Good use of Github flow ✔️


## Optional changes

- [ ] [OPTIONAL] Great effort creating the cards dynamically as suggested by the previous reviewer, But i would like you to work on the extra spaces in the pop up card in mobile view.
This wasn't raised by  the previous that is why i marked it optional, but it is highly recommended that you implement the changes so that you can have a wonderful project with a good user interface

mob-pop

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**